tests/output/ConsoleInteractionTest.py

```diff
@@ -439,8 +473,10 @@ def test_print_result_no_input(self):
                                  """
 Project wide:
 |    | [NORMAL] origin:
-|    | message
-""")
+|    | {}
+""".rstrip("\r\n").format(highlight("message", TextLexer(),
```

Comment:
why `rstrip()`? if you want to remove returns, just put `"""` onto the previous line :+1: